pages/products.html

```diff
@@ -0,0 +1,284 @@
+                    <button class="filter__clear">Clear</button>
+                </div>
+                <div class="filter__slider flex__column text__md ">
+                    <label for="slider">Select your Price Range</label>
+                    <div class="slider__price text__md font__light">
+                        <span>1000</span>
+                        <span>5500</span>
+                        <span>10000</span>
+                    </div>
+                    <input id="slider" type="range" min="1000" max="10000" value="5500" class="slider" />
+                    <div class="slider__output text__md"></div>
+                </div>
+                <div class="filter__category flex__column">
+                    <h5>Category</h5>
+
+                    <label><input type="checkbox">
+                        Action
+                    </label>
+                    <label><input type="checkbox">
+                        Arcade
+                    </label>
+                    <label><input type="checkbox">
+                        Strategy
+                    </label>
+                    <label><input type="checkbox">
+                        Casual
+                    </label>
+                    <label><input type="checkbox">
+                        Sports
+                    </label>
+                </div>
+                <div class="filter__rating flex__column">
+                    <h5>Rating</h5>
+
+                    <label>
+                        <input type="radio" name="rating">
+                        4 Stars and above
+                    </label>
+                    <label>
+                        <input type="radio" name="rating">
+                        3 Stars and above
+                    </label>
+                    <label>
+                        <input type="radio" name="rating">
+                        2 Stars and above
+                    </label>
+                    <label>
+                        <input type="radio" name="rating">
+                        1 Stars and above
+                    </label>
+                </div>
+                <div class="filter__sorting flex__column">
+                    <h5>Sort By</h5>
+                    <label>
+                        <input type="radio" name="sorting">
+                        Price: Low to High
+                    </label>
+                    <label>
+                        <input type="radio" name="sorting">
+                        Price: High to Low
+                    </label>
+                </div>
+            </aside>
+            <main class="products">
+                <div class="card card__shadow">
+                    <span class="card__icon right">
+                        <i class="far fa-heart"></i>
+                    </span>
+                    <span class="card__image">
+                        <img src="../assets/image.jpg" alt="card Image" />
```

Comment:
Suggestion: Can use a better alt description. Eg. product card or Mario card. Since it is a given that it is an image.

Reply:
Thanks, will change it.